ClemBot.Bot/bot/cogs/trivia_cog.py

```diff
@@ -0,0 +1,566 @@
+#Stephen Martin
+#5/9/2022
+#trivia_cog.py
+
+import json
+import logging
+import uuid
+import random
+import aiohttp
+import html
+import asyncio
+import typing as t
+from dataclasses import dataclass
+import discord
+import discord.ext.commands as commands
+from discord.ext.commands.errors import UserInputError
+import bot.bot_secrets as bot_secrets
+import bot.extensions as ext
+from bot.consts import Colors
+from bot.messaging.events import Events
+
+log = logging.getLogger(__name__)
+
+
+class TriviaCog(commands.Cog):
+
+    def cog_unload(self):
+
+        self.session.close()
+
+    def __init__(self, bot):
+        self.messages = {}
+        self.bot = bot
+        self.session = aiohttp.ClientSession(headers={'Connection': 'keep-alive'})
+
+    @ext.group(case_insensitive=True, invoke_without_command=True)
+    @ext.long_help(
+        "Use !trivia to return a random assortment of 10 trivia questions. React with emojis to submit your answer choice")
```

Comment:
we dont need to use the `!trivia` command invocation here becuase its handled by the example field